source/Nuke.Common/Tools/SignTool/SignToolSettings.cs

```diff
@@ -20,11 +21,25 @@ private static string GetToolPath()
 
         var platformIdentifier = EnvironmentInfo.Is64Bit ? "x64" : "x86";
         const string windowsKitLastVersion = "10";
+        const string windowsKitVersionWildcard = windowsKitLastVersion + ".*";
+        const string signtoolExe = "signtool.exe";
+        const string microsoftBuildToolsNugetPackage = "microsoft.windows.sdk.buildtools";
 
         var windowsKitsRootDirectory = programDirectory / "Windows Kits" / windowsKitLastVersion;
 
-        var signToolPath = windowsKitsRootDirectory.GlobFiles($"bin/{windowsKitLastVersion}.*/{platformIdentifier}/signtool.exe").LastOrDefault();
+        var signToolPath = windowsKitsRootDirectory.GlobFiles($"bin/{windowsKitVersionWildcard}/{platformIdentifier}/{signtoolExe}").LastOrDefault();
 
-        return signToolPath ?? windowsKitsRootDirectory.GlobFiles("App Certification Kit/signtool.exe").SingleOrDefault();
+        if(signToolPath == null)
+        {
+            var nugetPath = SettingsUtility.GetGlobalPackagesFolder(Settings.LoadDefaultSettings(null));
+
+            signToolPath = AbsolutePath.Create(nugetPath)
+                .GlobFiles($"{microsoftBuildToolsNugetPackage}/{windowsKitVersionWildcard}/bin/{windowsKitVersionWildcard}/{platformIdentifier}/{signtoolExe}")
```

Comment:
path structure for nuget packages is different, for example :
C:\Users\<user>\.nuget\packages\microsoft.windows.sdk.buildtools\10.0.20348\bin\10.0.20348\x64